docs/plugins/02-extra-plugins.md

```diff
@@ -228,6 +228,177 @@ Every plugin that works with Neovim works with LunarVim, here are some examples
   event = "BufRead",
```

Comment:
The `BufRead`  isn't particularly useful for lazy-loading, instead maybe use `cmd = "Octo"`

Also, don't you still need to call the setup function?